app/src/main/java/org/simple/clinic/storage/Migration_48_49.kt

```diff
@@ -0,0 +1,54 @@
+package org.simple.clinic.storage
+
+import android.database.Cursor
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.util.createUuid5
+import org.simple.clinic.util.toLocalDateAtZone
+import org.threeten.bp.Instant
+import org.threeten.bp.ZoneOffset
+
+@Suppress("ClassName")
+class Migration_48_49 : Migration(48, 49) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    val tableName = "BloodPressureMeasurement"
+
+    database.inTransaction {
+      //Using a random UUID to give a default value to non-null encounterUuid temporarily.
+      //The encounterUuid will be replaced with the correct value when the query executes in next line.
+      database.execSQL(""" ALTER TABLE $tableName ADD COLUMN "encounterUuid" TEXT NOT NULL DEFAULT '05f9c798-1701-4379-b14a-b1b18d937a33'  """)
```

Comment:
I don't understand why we are doing this - `05f9c798-1701-4379-b14a-b1b18d937a33` can you please explain?